scripts/release_scripts/get_repo_specific_changes.py

```diff
@@ -0,0 +1,159 @@
+# limitations under the License.
+
+"""Script that provides changes specific to oppia repo to be written
+to release summary file.
+"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+import python_utils
+from scripts import common
+
+GIT_CMD_DIFF_NAMES_ONLY_FORMAT_STRING = 'git diff --name-only %s %s'
+GIT_CMD_SHOW_FORMAT_STRING = 'git show %s:feconf.py'
+VERSION_RE_FORMAT_STRING = r'%s\s*=\s*(\d+|\.)+'
+FECONF_VAR_NAMES = ['CURRENT_STATE_SCHEMA_VERSION',
+                    'CURRENT_COLLECTION_SCHEMA_VERSION']
+FECONF_FILEPATH = os.path.join('', 'feconf.py')
+
+
+def get_changes_in_versions(release_tag_to_diff_against):
+    """Returns a list of schema version variable names in feconf that have
+    changed since the release against which diff is being checked.
+
+    Args:
+        release_tag_to_diff_against: str. The release tag to diff against.
+
+    Returns:
+        list(str). List of version variable names in feconf that changed.
+    """
+    changed_version_vars_in_feconf = []
+    git_show_cmd = (GIT_CMD_SHOW_FORMAT_STRING % release_tag_to_diff_against)
+    old_feconf = common.run_cmd(git_show_cmd.split(' '))
+    with python_utils.open_file(FECONF_FILEPATH, 'r') as feconf_file:
+        new_feconf = feconf_file.read()
+    for variable in FECONF_VAR_NAMES:
+        old_version = re.findall(
+            VERSION_RE_FORMAT_STRING % variable, old_feconf)[0]
+        new_version = re.findall(
+            VERSION_RE_FORMAT_STRING % variable, new_feconf)[0]
+        if old_version != new_version:
+            changed_version_vars_in_feconf.append(variable)
+    return changed_version_vars_in_feconf
+
+
+def _git_diff_names_only(first_release_tag, second_release_tag='HEAD'):
+    """Get names of changed files from git between two releases.
+
+    Args:
+        first_release_tag: str. The first release tag to diff against.
+        second_release_tag: str. The second release tag to diff against.
+
+    Returns:
+        list(str). List of filenames that are different between two releases.
+    """
+    diff_cmd = (
+        GIT_CMD_DIFF_NAMES_ONLY_FORMAT_STRING % (
+            first_release_tag, second_release_tag))
+    return common.run_cmd(diff_cmd.split(' ')).splitlines()
+
+
+def get_changes_in_setup_scripts(
+        release_tag_to_diff_against, only_include_changed_scripts=True):
+    """Returns a dict of setup script with a status of whether they have
+    changed or not since the release against which diff is being checked.
+
+    Args:
+        release_tag_to_diff_against: str. The release tag to diff against.
+        only_include_changed_scripts: bool. If true the output dict only
+            include the setup scripts which changed. If false, the output dict
+            will include the status for all setup scripts which are checked
```

Comment:
Be more explicit about what "which are checked" means -- do you just mean "all setup scripts"?

Reply:
Updated.